solr/server/solr/configsets/sample_techproducts_configs/conf/solrconfig.xml

```diff
@@ -84,6 +84,10 @@
   <lib dir="${solr.install.dir:../../../..}/contrib/velocity/lib" regex=".*\.jar" />
   <lib dir="${solr.install.dir:../../../..}/dist/" regex="solr-velocity-\d.*\.jar" />
 
+
+  <lib dir="${solr.install.dir:../../../..}/contrib/ltr/lib/" regex=".*\.jar" />
+  <lib dir="${solr.install.dir:../../../..}/dist/" regex="solr-ltr-\d.*\.jar" />
```

Comment:
The existing `<lib ...>` appear in alphabetical order i.e. perhaps preserve that when adding `ltr`.